lib/buildpacks-ci-pipeline-update-command.rb

```diff
@@ -6,6 +6,12 @@ def run!(concourse_target_name:, pipeline_name:, config_generation_command:, pip
     buildpacks_configuration = BuildpacksCIConfiguration.new
 
     pipeline_prefix = ENV.fetch('PIPELINE_PREFIX', '')
+    secrets_file = ENV.fetch('CONCOURSE_SECRETS_FILE', nil)
+
+    if secrets_file.nil?
```

Comment:
Do we care if the user sets `CONCOURSE_SECRETS_FILE` to `; rm -rf /*`? I guess they're shooting themselves in the foot there and don't care.